docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
```

Comment:
There didn't seem to be any reference documentation for any of the serializers.  I figured I'd put in the reference link like I did so that, if/when that does get added, the links will just start to work.  In the meantime, they'll show up in the markup so that they appear as class names.  It's not like a broken hyperlink gets created.  It just doesn't create a link for it at all.